CHANGELOG.md

```diff
@@ -1,6 +1,25 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [0.2.3-SNAPSHOT] - 2023-03-20
```

Comment:
Why 0.2.3-SNAPSHOT, when develop says 0.2.2-SNAPSHOT?
We still did not release any version, so you might add those changes in 0.2.2 (might also remove -SNAPSHOT, or we will remove (if not forgot) before creating release)

Reply:
At first, I thought that at each integration of the feature, the GHA will increment the latest number of version, so I thought that CHANGELOG should follow it, but I made a mistake, so now it's in 0.2.2. also, SNAPSHOT was removed.